src/components/ContentLayout/ContentBody/styles.ts

```diff
@@ -0,0 +1,9 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    flex: 1
```

Comment:
Add:
```
flexGrow: 1,
flexShrink: 0,
flexBasis: "auto",
```
this will make the "body" component fill the screen and the footer will always on the bottom

Reply:
`flex: 1` is a shorthand for 
```
flexGrow: 1,
flexShrink: 1,
flexBasis: "auto",
```
`flexShirt: 1` or `flexShrink: 0` doesn't really matter in this context
so I'll leace the `flex: 1` as is

Reply:
If it works for you, then I'm fine with it, but locally I had more space for the footer than was required